src/main.js

```diff
@@ -1 +1,173 @@
-alert("Testing script")
\ No newline at end of file
+//Using https://exchangerate.host/#/docs
+
+const $btnConsult = document.querySelector("#btn-consult");
+
+const $newConsultbtn = document.querySelector("#reset");
+
+handleSymbols();
+
+$btnConsult.onclick = validateForm;
+
+function validateForm(e) {
+  deletePreviousRates();
+  const $form = document.querySelector("#form");
+  const $title = document.querySelector("#title");
+  const $date = document.querySelector("#date").value;
+
+  if (isValidDate($date)) {
+    let date = document.querySelector("#date");
+    date.classList.remove("alert");
+    date.classList.remove("alert-danger");
```

Comment:
always read the docs first https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/remove